tests/test_wave_05.py

```diff
@@ -46,22 +47,24 @@ def test_get_goal(client, one_goal):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_get_goal_not_found(client):
-    pass
     # Act
     response = client.get("/goals/1")
     response_body = response.get_json()
 
-    raise Exception("Complete test")
+    # raise Exception("Complete test")
+    
     # Assert
-    # ---- Complete Test ----
-    # assertion 1 goes here
+    assert response.status_code == 404
+    
     # assertion 2 goes here
-    # ---- Complete Test ----
+    # added:
+    
+    assert response_body["message"] == "Goal 1 does not exist" 
```

Comment:
Nice work with this test completion! This assertion is a great way to validate functionality is working as expected